### DISCUSSION TITLE
Containers & Firefox

### DISCUSSION THREAD

**Discussion by SVilash** [mbnuqw/sidebery#1782] [General]
I was looking into a way to organize the abundance of tabs in Firefox and stumbled upon Sidebery the other day.
So far, it seems like the solution I've been looking for. I've organized the tabs I'm using on a daily basis and have categorized them in containers. However, I've noticed that when I relaunch Firefox and select 'restore previous session', all tabs are grouped back into the default group. The different containers i've created are still there but empty, so i have to organize them again. Is there any way around this?

I'm new to the Sidebery add-on, so apologies if this has been covered already. I did several searches before posting this, but coudn't find anyone with the same question.

**Comment by ArtofBugs** [comment]
For context, can you clarify what you mean by "containers" and "groups"?

"Group tabs" are special tabs specifically dedicated to grouping child tabs, like the top tab with the folder icon below:
![image](https://github.com/user-attachments/assets/77da955b-6683-499a-87d4-f777945f02a9)

The Sidebery settings page has a "containers" section, but it's related to [Firefox's containers functionality](https://support.mozilla.org/en-US/kb/how-use-firefox-containers), which doesn't seem to be what you're talking about.

I thought you could also be referring to "tab panels," which are the different sets of tabs you can switch between by clicking the icons at the top of the Sidebery sidebar - examples:
![image](https://github.com/user-attachments/assets/763466b8-e406-4ec0-99fc-4a90fe9a8d40)
![image](https://github.com/user-attachments/assets/5c016109-243e-4d1c-90fc-73c332779a66)
